.github/workflows/docker_publish.yml

```diff
@@ -0,0 +1,40 @@
+name: Build and publish Docker containers to GH registry.
+
+on:
+  push:
+    branches: [ master, develop ]
+  pull_request:
+    branches: [ master, develop ]
+
+env:
+  IMAGE_NAME_LIQUIDE_BLEIBEN: liquide-bleiben
+  IMAGE_NAME_CHATBOT: chatbot
+  TAGS: latest, ${{ github.GITHUB_SHA }}
+
+jobs:
+  liquide_bleiben_container:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Publish to Registry
+        uses: elgohr/Publish-Docker-Github-Action@2.14
+        with:
+          name: ${{ github.repository }}/liquide-bleiben
```

Comment:
I have some security concerns in regards to pushing the image with the codebeamer basic auth token build into it to the registry

Reply:
I would recommend to **NOT** publish the image with Codebeam credentials. See comment https://github.com/ChatBotCrew/liquide-bleiben/pull/11#discussion_r422140181